tutorials/kubernetes-auth-openid-rbac.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Kubernetes 1.6.1 Authentication via Google OpenID
+description: A sample process how to use Google Accounts with Kubernetes cluster with Role Based Access-Control (RBAC) authorization mode.
+author: miry
+tags: Kubernetes
+date_published: 2017-05-10
+---
+
+In this tutorial we will setup authentication and authorization using [RBAC] and [OpenID Connect]. [RBAC] introduced in the Kubernetes 1.6 [article](http://blog.kubernetes.io/2017/04/rbac-support-in-kubernetes.html). The article based on Micah Hausle’s [Reduce administrative toil with Kubernetes 1.3](https://www.skuid.com/blog/reduce-administrative-toil-with-kubernetes-1-3/)
```

Comment:
You don't need to link to RBAC a second time since there was one in the preceding sentence. However, the first time you mention it, you should also use its full name, i.e.:

"In this tutorial you will setup authentication and authorization using Role Based Access-Control ([RBAC](https://en.wikipedia.org/wiki/Role-based_access_control)) and [OpenID Connect](http://openid.net/connect/)."